### ISSUE TITLE
Add product to cart success message or error

### ISSUE THREAD

**Issue by prhost** [copona/copona#142]
When you add product to the cart, you either have no feedback, no success message or no error. It looks like it adds to the cart, not the php or js bug, it just updates the cart widget. I did not go any further into what happened.

**Comment by arnisjuraga** [comment]
Yes, i think its noticed. The issue is in Category/Search/or Product view? @janistravins can you look up this?

**Comment by prhost** [comment]
I only tested on home, when adding a direct product without accessing the internal one.

**Comment by prhost** [comment]
Inside the product and the listing of a category seems to work,

I do not know if there has to be with the same error, but the internal alerts of the product and category have different style.

![screen shot 2017-06-13 at 11 54 27](https://user-images.githubusercontent.com/3984760/27088944-747f1124-502f-11e7-9442-c7bdd29130b8.png)
![screen shot 2017-06-13 at 11 54 41](https://user-images.githubusercontent.com/3984760/27088945-748319a4-502f-11e7-95e4-afad643844fd.png)
